faqs/galaxy/updating_account_preferences.md

```diff
@@ -0,0 +1,21 @@
+---
+title: How to update account preferences?
+area: Account
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+1. Log into your existing account and update the email address under **User > Preferences**. DO NOT create another new account or it will be detected as a duplicate account. Any use of multiple accounts can result in data loss.
+
+2. If you instead want to start over with a new account, delete your old account(s) first under User > Preferences, then create a new account.
+
+Note: Updating a registered account email address will not prevent ongoing multiple/duplicate account detection.If you have more than one, existing or new or older, review below about how to clear that up. Use just one account and problems are avoided. Share data, not accounts.
```

Comment:
```suggestion
Note: Updating a registered account email address will not prevent multiple/duplicate account detection. If you have more than one, existing or new or older, review below about how to clear that up. Use just one account to avoid problems. Share data, not accounts.
```